tests/test_session.py

```diff
@@ -509,40 +511,32 @@ async def refresh_post(api_options: APIOptions, user_context: Dict[str, Any]):
     cookies = extract_all_cookies(response)
 
     assert "sAccessToken" in cookies
-    assert "sIdRefreshToken" in cookies
     assert "sRefreshToken" in cookies
 
     assert "anti-csrf" in response.headers
-    assert "id-refresh-token" in response.headers
     assert "front-token" in response.headers
 
     response = driver_config_client.post(
         "/auth/session/refresh",
         cookies={
             "sRefreshToken": cookies["sRefreshToken"]["value"],
-            "sIdRefreshToken": cookies["sIdRefreshToken"]["value"],
         },
         headers={"anti-csrf": response.headers["anti-csrf"]},
     )
 
     assert response.status_code == 401
     cookies = extract_all_cookies(response)
 
+    assert "anti-csrf" not in response.headers # TODO: This makes sense. But verify this
+    assert response.headers["front-token"] != ""
+
```

Comment:
This is inconsistent with the Node SDK. Is Node SDK behaviour correct? (What's the point of anti-csrf or front-token if accessToken and refreshToken are invalid)

See https://github.com/supertokens/supertokens-node/blob/d1c327d872042edc7c9320b6d67979041ef745ed/test/sessionExpress.test.js#L2989

Reply:
@porcellus should we change the node SDK for this?

Reply:
This is a fairly edge-case thing, only happening if the user throws inside the refresh override.
It doesn't cause a problem since the front token should be set to "remove". The anti-csrf token may be sent to/saved by the client, but it doesn't cause any issues. 
- We can change `clearSession` to remove the anti-csrf header from the response
- We could change the test to check the value of front-token